src/interactive/Toaster.jsx

```diff
@@ -0,0 +1,136 @@
+import CSSTransitionGroup from 'react-addons-css-transition-group';
+import PropTypes from 'prop-types';
+import React from 'react';
+import cx from 'classnames';
+
+export const DELAY_TIME = 3000;
+
+/**
+ * @module Toaster
+ */
+class Toaster extends React.PureComponent {
```

Comment:
I think this class is probably going to need to be refactored, although it might require some testing to confirm. 

The main issue is that it currently expects _all_ toasts to be provided as `props` when the component is _first mounted_ - if new toasts arrive as new props later in the component's lifecycle, they will be ignored. You'll need to use `componentWillReceiveProps` to update `this.state.toasts` in some way